doc/memo_interpolation.mdwn

```diff
@@ -1,26 +1,25 @@
 # Spectral template interpolation of AMBRE
-Stellar model templates for flux calibration are prepared based on the AMBRE template set. To obatain models at smaller parameter steps than the original ones of AMBRE, the AMBRE templates are interpolated along model parameters (effective temperature, surface gravity, metallicity, and alpha element index) using the radial basis function. 
+Stellar model templates for flux calibration are prepared based on the AMBRE template set. To obtain models at smaller parameter steps than the original ones of AMBRE, the AMBRE templates are interpolated along model parameters (effective temperature, surface gravity, metallicity, and alpha element index) using the radial basis function. 
 
 ## AMBRE
 <!-- ### About AMBRE -->
-AMBRE is the high resolution stellar model templates (de Laverny et al. 2012, A&A, 544, 126). It covers a wavelength range from 300 nm to 1200 nm. The original resolution is 0.001 nm/pix. There are four parameters of Teff, log(g), Z, [alphe/Fe]. 
+AMBRE is a high resolution stellar model templates (de Laverny et al. 2012, A&A, 544, 126). The wavelength coverage is from 300 nm to 1200 nm. The original resolution is 0.001 nm/pix. There are four parameters of Teff, log(g), Z, [alpha/Fe]. 
 
 <!-- ### Input data for interpolation -->
 Out of the AMBRE templates, 3190 model templates with Teff = 5500 - 8000 K and plane-parallel atmosphere are used as input data of the interpolation.
-The wavelength pixel of the input templates are binned into 0.01 nm/pix to reduce a number of calculation.
-They are stored as a FITS format file of 0.4 MB size. In total, 1.27 GB.
-
+The spectral resolution is reduced down to 0.01 nm/pix by pixel binning to reduce a number of calculation.
```

Comment:
[...] to reduce the number of calculations [...]

Reply:
done.